hugo/layouts/partials/navbar.html

```diff
@@ -21,14 +21,20 @@
 				{{ $baseurl := urls.Parse $.Site.Params.Baseurl }}
 				{{ if eq .Name "Documentation" }}
 				<div class="dropdown">
-					<a href="/docs" class="nav-link">Documentation</a>
+					<a href="{{ with .Page }}{{ .RelPermalink }}{{ else }}{{ .URL | relLangURL }}{{ end }}" class="nav-link">Documentation</a>
 					<div class="dropdown-content">
-					  <a class="taxonomy-term" href="/docs">Users</a>
-					  <a class="taxonomy-term" href="/docs">Operators</a>
-					  <a class="taxonomy-term" href="/docs">Developers</a>
-					  <a class="taxonomy-term" href="/docs">All</a>
+					  <a class="taxonomy-term" href="{{ with .Page }}{{ .RelPermalink }}{{ else }}{{ .URL | relLangURL }}{{ end }}">Users</a>
+					  <a class="taxonomy-term" href="{{ with .Page }}{{ .RelPermalink }}{{ else }}{{ .URL | relLangURL }}{{ end }}">Operators</a>
+					  <a class="taxonomy-term" href="{{ with .Page }}{{ .RelPermalink }}{{ else }}{{ .URL | relLangURL }}{{ end }}">Developers</a>
+					  <a class="taxonomy-term" href="{{ with .Page }}{{ .RelPermalink }}{{ else }}{{ .URL | relLangURL }}{{ end }}">All</a>
 					</div>
 				</div>
+				{{ else if eq .Name "Join us on"}}
+				<a href="{{ with .Page }}{{ .RelPermalink }}{{ else }}{{ .URL | relLangURL }}{{ end }}" {{ if ne
+					$url.Host $baseurl.Host }}target="_blank" {{ end }} class="btn btn-primary slack-button">
+					<span>{{ .Name }}</span>
+					<img src="images/slack-icon.svg" alt="Slack">
```

Comment:
You could replace this with a template using `relUrl` and the icon will show up everywhere 😄 
Before you do that, could you move the svg to `images/branding/` for consistency in using such icons.
It should look something like this:
```suggestion
					<img src='{{"images/branding/slack-icon.svg" | relURL}}' alt="Slack">
```

Reply:
Thank you for the help! The changes have been made.